bin/cohttp_server_lwt.ml

```diff
@@ -89,6 +81,30 @@ let handler ~info ~docroot ~verbose ~index (ch,conn) req body =
   | e -> fail e
   )
 
+let handler ~info ~docroot ~verbose ~index (ch,conn) req body =
+  let uri = Cohttp.Request.uri req in
+  let path = Uri.path uri in
+  (* Log the request to the console *)
+  printf "%s %s %s %s\n%!"
+    (Cohttp.(Code.string_of_method (Request.meth req)))
+    path
+    (match verbose with
```

Comment:
I have no idea. It's been there since <https://github.com/dsheets/ocaml-cohttp/blob/00fbdbebe6c6f30c832615fd52b478c41e0e8710/bin/cohttp_server_lwt.ml> (first commit of this code) and should probably be cleaned up. I think there was some idea about verbose/non-verbose console logging but it never really materialized.